docs/src/views/IntegrateCustomComponents.tsx

```diff
@@ -0,0 +1,238 @@
+import * as React from 'react'
+import * as cx from 'classnames'
+import { NavLink } from 'react-router-dom'
+import { Header } from 'semantic-ui-react'
```

Comment:
I think that we can use own `Header` component there 😃

Reply:
Switching the theme to dark will make the Header component white, and we don't have the other themes implemented for all page (like making the background dark in dark-mode). I would not like to introduce unnecessary providers just for handling things like that. In all docs pages, the Headers are from semantic-ui, so when the dark mode will be implemented for all page, we can safely replace them all in one iteration.